apps/vercel/src/components/common/VercelIcon.tsx

```diff
@@ -0,0 +1,12 @@
+export const VercelBrand = {
+  primaryColor: '#000',
+  url: '',
+};
+
+export default function AmplifyIcon(props: any) {
```

Comment:
maybe add a TODO (If this is in fact the amplify icon) or rename (if it's just misnamed)